### PR TITLE
Update the executions model

### PR DESCRIPTION
**Pull request by anandswaminathan**

### CODE REVIEW

File: pkg/repositories/config/migrations.go [hunk 1]
```diff
@@ -109,4 +109,11 @@ var Migrations = []*gormigrate.Migration{
 			return tx.Exec("update node_executions set parent_task_execution_id = NULL where parent_task_execution_id = 0").Error
 		},
 	},
+	// Update executions table to add cluster
+	{
+		ID: "2019-09-27-executions",
```
Comment (katrogan) [inline pkg/repositories/config/migrations.go]: can you update the id in line 59 instead?

Comment (katrogan) [reply pkg/repositories/config/migrations.go]: very well, can you add a rollback command that drops the new column?

File: pkg/repositories/config/migrations.go [hunk 2]
```diff
@@ -109,4 +109,14 @@ var Migrations = []*gormigrate.Migration{
...
+		Migrate: func(tx *gorm.DB) error {
+			return tx.AutoMigrate(&models.Execution{}).Error
+		},
+		Rollback: func(tx *gorm.DB) error {
+			return tx.AutoMigrate(&models.Execution{}).Error
```
Comment (wild-endeavor) [inline pkg/repositories/config/migrations.go]: @katrogan gorm deletes columns too?

Comment (katrogan) [reply pkg/repositories/config/migrations.go]: nope, this rollback function is identical to the migrate. i think you need to explicitly drop the new column.

**Pull request merged**